### PR TITLE
Accessibilty fix sections nitin

### PR DESCRIPTION
**Pull request by NitinCopods**
- Sections Page : Bug fixes & Accessibility issue -
1 : In Add Question, Check order toggle button is visible while there is only one option.
2 : In Add Question, Option Input field in Add Question is not aligned.
3 : In Section Details Card, Title Border is not visible when toggling the questions below.
4 : In Section Detail, toggling question doesn't works when clicking on the card 
it works only when clicking the icon.
5 : In Sections Page, Sorting Icon doesn't have tabIndex.
6 : In Section Details, Add Question button is wrapped inside a link tag creating 2 tab indexes on the
same UI.
7 : In Sections page, Outline is not coming properly on focus on section list card, sorting icon, filter dropdown.

### CODE REVIEW

File: app/components/sections/QuestionCard.tsx [hunk 1]
```diff
@@ -17,9 +17,15 @@ const QuestionCard = ({
   return (
     <div
       key={question.id}
-      className="flex flex-col rounded-lg border border-gray-300 bg-gray-50 px-6 py-7"
+      className="flex flex-col rounded-lg border border-gray-300 bg-gray-50 px-6 py-7 cursor-pointer"
+      onClick={() => {
+       onAccordianToggle(
+         isExpanded == -1 ? index : isExpanded == index ? -1 : index
```
Comment (richa-copods) [inline app/components/sections/QuestionCard.tsx:23]: Please use strict check `===`

File: app/components/sections/Sections.tsx [hunk 2]
```diff
@@ -68,7 +70,7 @@ const Sections = ({
 
       {/* list */}
       <div
-        className="flex flex-1 flex-col gap-6 overflow-auto"
+        className="flex flex-1 flex-col gap-6 overflow-auto p-2-px"
         id="section-cards"
       >
         {sections?.map((section: any) => (
```
Comment (richa-copods) [inline app/components/sections/Sections.tsx:76]: Please give proper type

File: app/components/tests/ChipGroup.tsx [hunk 3]
```diff
@@ -35,11 +35,11 @@ const ChipGroup = ({
           >
             <Menu.Items
               id="menu-items"
-              className="section-menu absolute z-40 max-h-52 overflow-auto rounded-md bg-white py-4 px-4 shadow-lg ring-1 ring-black ring-opacity-5 focus:outline-none"
+              className=" absolute z-40 max-h-52 overflow-auto rounded-md bg-white py-4 px-4 shadow-lg ring-1 ring-black ring-opacity-5 focus:outline-none"
             >
               {sections.map((sect, i) => {
                 return (
-                  <div className=" flex flex-col px-2 py-2" key={i}>
+                  <div className="section-menu flex flex-col px-2 py-2" key={i}>
```
Comment (richa-copods) [inline app/components/tests/ChipGroup.tsx:42]: Please give proper key maybe section id, avoid giving index as key

File: cypress/e2e/group-by-test/groupByTestTable.cy.ts [hunk 4]
```diff
@@ -97,8 +100,6 @@ describe('Test for GroupByTestTable, Result', () => {
     cy.get('#2').find('hr').should('have.class', 'bg-primary')
     cy.get('button#submitButton').should('have.text', cypress.submit).click()
     cy.location('pathname', { timeout: 60000 }).should('include', '/tests')
-    cy.get('a').find('#Group_By_Tests').should('have.text', cypress.results).click()
-    cy.location('pathname', { timeout: 60000 }).should('include', '/results')
   })
 
   let value: any
```
Comment (richa-copods) [inline cypress/e2e/group-by-test/groupByTestTable.cy.ts:105]: Please give proper type

File: tailwind.config.js [hunk 5]
```diff
@@ -31,6 +31,9 @@ module.exports = {
       },
       gridTemplateColumns: {
         18: 'repeat(18, minmax(0, 1fr))'
+      },
+      padding:{
+        "2-px":'2px'
```
Comment (richa-copods) [inline tailwind.config.js:36]: No need to create this, could use this- `p-0.5  /* padding: 0.125rem;  2px */`
https://tailwindcss.com/docs/padding

**Review by richa-copods — DISMISSED** [review]

File: app/components/sections/add-question/OptionForQuestion.tsx [hunk 6]
```diff
@@ -123,7 +123,8 @@ export default function OptionForQuestion({
             {addQuestion.createOptions}
           </div>
         ) : (
-          <div className="flex items-center justify-center gap-2 text-base font-medium leading-6 text-gray-600">
+          <div
+            className={`flex items-center justify-center gap-2 text-base font-medium leading-6 text-gray-600 ${textCorrectAnswer.length < 2 && 'pointer-events-none' }`}>
```
Comment (yash6102000) [inline app/components/sections/add-question/OptionForQuestion.tsx:127]: no need to use leading , when we use tailwind font class..itll apply line height by default

File: tailwind.config.js [hunk 7]
```diff
@@ -31,6 +31,9 @@ module.exports = {
       },
       gridTemplateColumns: {
         18: 'repeat(18, minmax(0, 1fr))'
+      },
+      padding:{
+        "0.5":'0.125rem'
```
Comment (yash6102000) [inline tailwind.config.js:36]: why we are using custom padding here..tailwind privide p-0.5

File: app/components/sections/add-question/OptionForQuestion.tsx [hunk 8]
```diff
@@ -138,6 +138,7 @@ export default function OptionForQuestion({
             {addQuestion.createOptions}
           </div>
         ) : (
+
```
Comment (arpita-20) [inline app/components/sections/add-question/OptionForQuestion.tsx:141]: please remove extra spaces

File: app/components/sections/QuestionCard.tsx [hunk 9]
```diff
@@ -36,6 +36,7 @@ const QuestionCard = ({
       }}
     >
       <div className="items-top break flex justify-between text-base text-gray-600">
+
```
Comment (arpita-20) [inline app/components/sections/QuestionCard.tsx:39]: remove. extra spaces

File: cypress/e2e/section-tests/Section.cy.ts [hunk 10]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (arpita-20) [inline cypress/e2e/section-tests/Section.cy.ts:1]: code should always start from first line. Remove extra space

File: cypress/e2e/smoke.cy.ts [hunk 11]
```diff
@@ -95,6 +96,7 @@ describe('smoke tests', () => {
         cy.get('[data-cy="submit"]').click()
       })
   })
+
```
Comment (arpita-20) [inline cypress/e2e/smoke.cy.ts:99]: remove extra space in this file

**Review by anuragP04 — APPROVED** [review]

**Pull request merged**